lesson6/method isdigit.py

```diff
@@ -0,0 +1,22 @@
+def analyze(number):
+    for i in number:
+        value = i.isdigit()
+        if value is True:
+            print(f"Вы ввели положительное целое число: {i}")
+        elif value is False:
+            try:
```

Comment:
It is good you use `try except` here.. but we haven't gone through this material yet. Lets think how can we make it without `try except`